packages/visual-editor/src/components/puck/Phone.tsx

```diff
@@ -96,8 +100,7 @@ const Phone: React.FC<PhoneProps> = ({ phone, format, includeHyperlink }) => {
   );
 };
 
-const PhoneComponent: ComponentConfig<PhoneProps> = {
-  label: "Phone",
```

Comment:
We should keep the label for all Puck components. Some have it and some don't. I know the label is automatically set to the name of the component, but let's set it on all of them for consistency and clarity.